Projects/LinkedListIterator/LinkedListIterator/LinkedListIterator/SLList.h

```diff
@@ -230,6 +229,20 @@ class TSLList
 		}	
 	}
 
+	//-------------------------------------------------------------------------------------------
+	void RemoveHead()
+	{
+		FNode* NewHead = Head->Next;
+		delete Head;
+		Head = NewHead;
+
+		if (Size == 1)
+		{
+			Tail = nullptr;
+		}
+		--Size;
+	}
+
```

Comment:
LOL! I can´t comment from this part down, btw, just a comment about the clear method, you should call RemoveHead() inside this function, maybe it is easier to read since you created the method.